pkg/pillar/cmd/loguploader/loguploader.go

```diff
@@ -42,6 +42,8 @@ const (
 
 	keepSentDir = types.NewlogKeepSentQueueDir
 
+	backoffMaxUploadIntv   = 300
+	backoffMaxTimeout      = 3600 // backoff timeout in one hour
```

Comment:
timeout of what? Is this the maximum time during which we do exponential backoff before going back to square one? Please add some comments explaining this in the appropriate places.

Reply:
ok, will put more comments. this timeout is that before this timeout expires, we don't check normal upload schedule, we only use the backoff current interval setting.